### DISCUSSION TITLE
Invert direction?

### DISCUSSION THREAD

**Discussion by gcormier** [gin66/FastAccelStepper#303] [General]
Is there a way to invert step direction? If not, would be great if it could be added to the `setDirectionPin()` perhaps?

**Comment by gin66** [comment]
Thanks for the proposal. And in the moment, changing the polarity of the step pin is not possible. As this would need 
 to be implemented on driver level: which uC are you thinking of ?

**Reply by gcormier** [reply]
I'm using an ESP32-S3 and TMC2209.

**Reply by gin66** [reply]
For >=idf5.3 only the new rmt-driver is usable, so the effort is moderate. For rmt-driver only few changes are needed to get this done (init-value, completion value and all symbols xor‘d to toggle 0/1). Plus an API call.

Question is who can do it when. Currently I have not enough time to do that change. PRs are welcome.
